scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -586,6 +588,36 @@ class ScioContext private[scio] (val options: PipelineOptions,
     }
 
   /**
+    * Get an SCollection for a CloudSql query
+    * @group input
+    */
+  def cloudSqlSelect[T: ClassTag](readOptions: JdbcReadOptions[T]): SCollection[T] = {
```

Comment:
How is this different from `jdbcSelect`? This should probably expose fewer options relevant to CloudSQL?